SQLQuery_3(homework).sql

```diff
@@ -0,0 +1,303 @@
+CREATE DATABASE ZooManagement;
+
+USE ZooManagement;
+
+CREATE TABLE ZooInstitutions (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Name VARCHAR(100) NOT NULL, 
+    City VARCHAR(150) DEFAULT 'Country Romania',
+);
+
+
+CREATE TABLE Visitors (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfDay VARCHAR(50) NOT NULL,
+    FirstName VARCHAR(100) NOT NULL, 
+    LastName VARCHAR(100) NOT NULL,
+    ZooId INT REFERENCES ZooInstitutions(Id)
```

Comment:
Because ZooId is referenced in the Tickets table as well it becomes redundant here. Basically you can find out to which zoo a visitor went by looking at his ticket.